atlasdb-dbkvs/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/TestConfigLoading.java

```diff
@@ -36,7 +36,11 @@
 public class TestConfigLoading {
     @Test
     public void testLoadingConfig() throws IOException {
-        AtlasDbConfigs.load(new File(getClass().getClassLoader().getResource("postgresTestConfig.yml").getFile()));
+        // Palantir internal runs this test from the jar rather than from source. This means that the resource
```

Comment:
Nit: This comment + loading config code seems to be reused below.  Could be worth extracting a `private AtlasDbConfig loadConfig()` method.